test/unit_test/cli/test_cli.py

```diff
@@ -333,7 +334,7 @@ def test_shared_cache_delete_all_with_confirmation(mock_AzureContainerClientFact
 @patch("huggingface_hub.repo_exists")
 def test_quantize_command(mock_repo_exists, mock_run, algorithm_name, tmp_path):
     # setup
-    output_dir = tmp_path / "output_dir"
+    output_dir = "output_dir"
```

Comment:
Why change this?

Reply:
why do you change `output_dir = tmp_path / "output_dir"` to `output_dir = "output_dir"`